buildout.cfg

```diff
@@ -1,8 +1,15 @@
 [buildout]
+extends = https://raw.githubusercontent.com/zopefoundation/groktoolkit/resurrection-python3/grok.cfg
```

Comment:
This cfg file will end up on `https://raw.github.com/zopefoundation/groktoolkit/master/grok.cfg` at some point?

Reply:
yes